rest-mvc/src/main/java/com/intuit/tank/rest/mvc/rest/controllers/AgentController.java

```diff
@@ -122,62 +134,62 @@ public ResponseEntity<CloudVmStatus> getInstanceStatus(@PathVariable @Parameter(
         if (status != null) {
             return new ResponseEntity<CloudVmStatus>(status, HttpStatus.OK);
         }
-        return new ResponseEntity<CloudVmStatus>(status, HttpStatus.NOT_FOUND);
```

Comment:
@Zakaria-Kofiro Not seeing a service call here, is that expected?

Reply:
Oh it calls `agentService.getInstanceStatus(instanceId)` before the null check

Reply:
Oh yeah, I am blind :O